assembl/indexing/settings.py

```diff
@@ -81,6 +81,7 @@ def add_index_languages(props, names):
         'creation_date': DATE,
         'id': LONG,
         'parent_id': {'type': 'long', 'null_value': 0},
+        'idea_id': LONG,
```

Comment:
A message can be associated to several ideas, so type should probably not be LONG but an array of LONG, no? And so idea_id would not be the best name, rather something like ideas_ids

Reply:
With the elasticsearch mapping syntax, there is no array per se. Declaring here idea_id as long can contain long or an array of long.